apps/chat-e2e/src/assertions/baseAssertion.ts

```diff
@@ -139,10 +139,40 @@ export class BaseAssertion {
       .toHaveText(expectedText);
   }
 
+  public async assertElementAttribute(
+    element: BaseElement | Locator,
+    attribute: string,
+    expectedValue: string,
+    expectedMessage?: string,
+  ) {
+    const elementLocator =
+      element instanceof BaseElement
+        ? element.getElementLocator()
+        : (element as Locator);
+    await expect
+      .soft(
+        elementLocator,
+        expectedMessage ?? ExpectedMessages.elementAttributeValueIsValid,
+      )
+      .toHaveAttribute(attribute, expectedValue);
+  }
+
   public async assertElementColor(element: BaseElement, expectedColor: string) {
```

Comment:
Same here, since I see it - Can element accept either a BaseElement or a Locator?

Reply:
Since `getComputedStyleProperty()` function is available only for `BaseElement` class, it is not possible to use double types in the method signature